sdk/ml/azure-ai-ml/azure/ai/ml/_schema/component/command_component.py

```diff
@@ -35,6 +36,7 @@ class Meta:
             NestedField(MPIDistributionSchema, unknown=INCLUDE),
             NestedField(TensorFlowDistributionSchema, unknown=INCLUDE),
             NestedField(PyTorchDistributionSchema, unknown=INCLUDE),
+            NestedField(RayDistributionSchema, unknown=INCLUDE),
```

Comment:
Consider use ExperimentalField for preview features. More info can be found here.

https://msdata.visualstudio.com/Vienna/_git/sdk-cli-v2?path=/docs/SDK_preview_features.md&_a=preview